public/walkthroughs/1/walkthrough.adoc

```diff
@@ -0,0 +1,357 @@
+
+A more structured approach involves:
+
+* a simple integration that developers can understand intuitively
+* simple provisioning and configuration
+* built-in monitoring
+
+This walkthrough describes how  to create such an integration, using OpenShift, {messaging-service} and API connectors in Fuse.
+
+++++
+<img src="/images/wt1.png" class="img-responsive" alt="integration">
+++++
+
+// end::master-1[]
+
+[time=5]
+[id='integrating-eventdriven-and-apidriven-applications']
+[id='creating-connections']
+== Creating connections
+:context: creating-connections
+
+// tag::creating-api-connector[]
+
+[id='creating-api-connector_{context}']
+[.integr8ly-docs-header]
+=== Creating an API Connector 
+
+. Log in to the link:{fuse-url}[Red Hat Fuse Online, window="_blank"] console.
+
+. Select *Customizations* from the left hand menu.
+
+. Select the *Create API Connector* button to start the *API Client Connector* wizard.
+
+. When prompted to *Upload OpenAPI Specification*, select *Use a URL*:
+.. Enter the following in the URL field:
++
+[subs="attributes+"]
+---- 
+http://{spring-boot-url}/v2/api-docs
+----
+
+.. Click *Next*.
+
+. When prompted with *Review Actions*, select *Next*.
+
+. When prompted with *Specify Security*, select *Next*.
+
+. When prompted with *Review/Edit Connector Details*:
+.. Enter the following in the *Connector Name* field:
++
+[subs="attributes+"]
+----  
+Walkthrough {walkthrough-id} CRUD Connector
+----
+
+.. Click *Create API Connector*.
+
+[type=verification]
+Check the *Customizations* screen of the link:{fuse-url}[Red Hat Fuse Online, window="_blank"] console to make sure the new connector exists.
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+
+// end::creating-api-connector[]
+
+// tag::creating-amqp-connection-in-fuse[]
+
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+:enmasse: Red Hat AMQ Online
+
+[id='creating-amqp-connection-in-fuse_{context}']
+[.integr8ly-docs-header]
+=== Creating AMQP connection in Red Hat Fuse Online
+
+To receive messages from EnMasse, you create a connection in Red Hat Fuse Online.
+
+
+:fuse-url: https://eval.apps.city.openshiftworkshop.com/
+:openshift-url: https://master.city.openshiftworkshop.com/console/project/eval/overview
+
+. Log in to the link:{fuse-url}[Red Hat Fuse Online, window="_blank"] console.
+
+. Select *Connections* from the left hand menu.
+
+. Select the *Create Connection* button to start the *Create Connection* wizard.
+
+. Select *AMQP* to configure an *AMQP Message Broker* connection.
+
+. Enter the connection URI relating to {enmasse}:
++
+[subs="attributes+"]
+----  
+amqp://{messaging-broker-url}:5672?amqp.saslMechanisms=PLAIN
+----
+
+. Enter the username relating to {enmasse}:
++
+[subs="attributes+"]
+----  
+{messaging-username}
```

Comment:
This'd be named `amq-credentials-username` now. See the list of attribute names in https://github.com/integr8ly/tutorial-web-app/blob/add-own-walkthroughs/public/walkthroughs/my-custom-walkthrough/walkthrough.adoc